app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,169 @@ package org.sil.storyproducer.model
         val filename = when (phaseType){
-            PhaseType.DRAFT -> Workspace.activeStory.slides[slideNum].narrationFile
-            PhaseType.COMMUNITY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.CONSULTANT_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.DRAMATIZATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.BACKT -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.TRANSLATE_REVISE -> Workspace.activeStory.slides[slideNum].narrationFile
+            PhaseType.COMMUNITY_WORK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.ACCURACY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.VOICE_STUDIO -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.BACK_TRANSLATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
             else -> ""
         }
         return Story.getFilename(filename)
     }
 
-    fun getPrettyName() : String {
+    /**
+     * get displayable name for a phase (used to create option menu help dialog and logging)
+     * @return String
+     */
+    fun getDisplayName() : String {
         return when (phaseType) {
             PhaseType.LEARN -> "Learn"
-            PhaseType.DRAFT -> "Translate"
-            PhaseType.CREATE -> "Finalize"
+            PhaseType.TRANSLATE_REVISE -> "Translate + Revise"
+            PhaseType.COMMUNITY_WORK -> "Community Work"
+            PhaseType.ACCURACY_CHECK -> "Accuracy Check"
+            PhaseType.VOICE_STUDIO -> "Voice Studio"
+            PhaseType.FINALIZE -> "Finalize"
             PhaseType.SHARE -> "Share"
-            PhaseType.COMMUNITY_CHECK -> "Community Work"
-            PhaseType.CONSULTANT_CHECK -> "Accuracy Check"
+            PhaseType.BACK_TRANSLATION -> "Back Translation"
             PhaseType.WHOLE_STORY -> "Whole Story"
             PhaseType.REMOTE_CHECK -> "Remote Check"
-            PhaseType.BACKT -> "Back Translation"
-            PhaseType.DRAMATIZATION -> "Voice Studio"
             else -> phaseType.toString().toLowerCase()
         }
     }
 
-    fun getDisplayName() : String {
+    /**
+     * get name used for storing audio files
+     * @return String
+     */
+    fun getFileSafeName() : String {
         return when (phaseType) {
-            PhaseType.DRAFT -> "Translation Draft"
-            PhaseType.COMMUNITY_CHECK -> "Comment"
-            PhaseType.CONSULTANT_CHECK -> "Accuracy"
-            PhaseType.WHOLE_STORY -> "Whole"
-            PhaseType.REMOTE_CHECK -> "Remote"
-            PhaseType.BACKT -> "BackTrans"
-            PhaseType.DRAMATIZATION -> "Studio Recording"
-            PhaseType.CREATE -> "Finalize"
+            PhaseType.LEARN -> "Learn"
+            PhaseType.TRANSLATE_REVISE -> "TranslateRevise"

```

Comment:
Whoa! Does this mean that a new build can't read the audio files saved by past app versions (v2.3.x or v3.0.x)?
That would break our current commitment to be able to read existing projects.
If this concern is accurate, I would suggest that you extract your code changes that alter reading/writing of audio file names, and put them as comments in a separate branch. In that new branch we can write the code so that it can read old or new filenames.